serializr.d.ts

```diff
@@ -71,8 +71,9 @@ export function map(propSchema: PropSchema): PropSchema;
 
 export function mapAsArray(propSchema: PropSchema, keyPropertyName: string): PropSchema;
 
-export function custom(serializer: (value: any) => any, deserializer: (jsonValue: any) => any): PropSchema;
+export function custom(serializer: (value: any) => any, deserializer: (jsonValue: any, context?: any, oldValue?: any) => any): PropSchema;
+export function custom(serializer: (value: any) => any, deserializer: (jsonValue: any, context: any, oldValue: any, callback: (err: any, result: any) => void) => any): PropSchema;
```

Comment:
An empty line break for readability?

Reply:
currently, line breaks are only between different functions, not signature alternatives. We just introduced the latter with the final version.

Reply:
👍 Your explanation is reasonable.